app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenter.kt

```diff
@@ -43,4 +43,5 @@ public interface MainPresenter {
     public fun onReroute(location: Location)
     public fun onMapMotionEvent(): Boolean
     public fun onLongPressMap(event: MotionEvent): Boolean
+    open fun onExitNavigation()
```

Comment:
Is `open` redundant on an interface? My (limited) understanding is that `open` allows overriding, which is sort of built into an interface.